app/models/statement_item.rb

```diff
@@ -3,7 +3,7 @@ class StatementItem < ApplicationRecord
   REFUNDABLE_STATES = %w[awaiting_clawback clawed_back].freeze
 
   belongs_to :statement
-  # belongs_to :declaration
+  belongs_to :declaration
```

Comment:
I think we'll want the inverse relationships on declaration as well as we often look through the declaration to the statement/line items

Reply:
I was thinking the `has_many` etc can be added as needed

Reply:
Fair enough 👌